### DISCUSSION TITLE
AttributeError: 'OpenAI' object has no attribute 'total_usage_summary'

### DISCUSSION THREAD

**Discussion by cbdonohue** [microsoft/autogen#1609] [Q&A] [answered]
I'm trying to run `notebook/agentchat_dalle_and_gpt4v.ipynb` from HEAD
but I get the error
`AttributeError: 'OpenAI' object has no attribute 'total_usage_summary'
`

**Comment by ekzhu** [comment, marked as answer]
Looks like a bug due to changes in the OpenAI client. @olgavrou for awareness. 

You are welcome to submit a pr to fix this

**Reply by cbdonohue** [reply]
Thanks, I'll take a look into fixing it.
